playbook.md

```diff
@@ -712,41 +712,49 @@ The guide starts with the user research workflow, which describes the things tha
 
 Designers at dxw follow these principles:
 
-1. **Have faith in your instinct and experience**
+1. **Democratise the design process**
 
-   Don’t throw away your gut instinct, but do filter decisions through the lens of research and data.
+   Take into account, and value highly, the input of others in the design process. While we are ultimately responsible for design in a delivery team, we don’t work in isolation.
```

Comment:
Maybe we should follow GDS guidelines and not use negative contractions? For example this would become `we do not work in isolation`